### PR TITLE
Cypress/E2E: Promote and Update

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary

Promoted ```custom_emoji_2_spec.js``` to prod
Added unstable test case to ```custom_emoji_3_spec.js``` 
Fixed ```MM-T1391 Verify the manage options displayed for Guest User```

### CODE REVIEW

**Review by josephbaylon — CHANGES_REQUESTED** [review]
@furqanmlk Please fix lint issues

**Review by josephbaylon — APPROVED** [review]
LGTM

**Review by prapti — CHANGES_REQUESTED** [review]
Hi Furqan,
I'm not too sure about promoting `custom_emoji_3` yet. In the last 5 runs, looks like the only test that's been stable is T2183.
T2184 has failed in 2 runs (builds 4061 and 3972).
T2185 has failed in 2 runs (builds 4012 and 3948).
Should we wait on promoting that spec until we take a deeper look?

**Comment by furqanmlk** [comment]
Hi @prapti, 
I think you misunderstood. 
custom_emoji_3_spec.js did get promoted. I added unstable test cases to this spec.

> Hi Furqan,
> 
> I'm not too sure about promoting `custom_emoji_3` yet. In the last 5 runs, looks like the only test that's been stable is T2183.
> 
> T2184 has failed in 2 runs (builds 4061 and 3972).
> 
> T2185 has failed in 2 runs (builds 4012 and 3948).
> 
> Should we wait on promoting that spec until we take a deeper look?

**Comment by prapti** [comment]
Oh sorry @furqanmlk I meant `custom_emoji_2` (not `custom_emoji_3`). Since you're promoting that spec, and only one test in that file (T2183) seems stable but not the other two (T2184 which is still in that file, and T2185 which you're moving the unstable `custom_emoji_3`) are both quite unstable.

**Review by saturninoabril — APPROVED** [review]
Not sure if moving around test case (like `MM-T2185`) is something that we should normally do. In my mind, move test either a passing/failing test into a new spec file to isolate but a fix should be made to the failing test on the same PR, unless such failed test is a known issue like product bug, or any limitation in our test infra.

I'm good as is but open for discussion as a group.

File: e2e/cypress/integration/emoji/custom_emoji_2_spec.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+
```
Comment (furqanmlk) [inline e2e/cypress/integration/emoji/custom_emoji_2_spec.js:10]:
```suggestion
```

**Comment by furqanmlk** [comment]
> Oh sorry @furqanmlk I meant `custom_emoji_2` (not `custom_emoji_3`). Since you're promoting that spec, and only one test in that file (T2183) seems stable but not the other two (T2184 which is still in that file, and T2185 which you're moving the unstable `custom_emoji_3`) are both quite unstable.

Thanks @prapti 
Yes you are right, I moved T2183 to wrong file. 
I have moved T2183 to `custom_emoji_1` as it is now stable, rest I did not change.

**Comment by furqanmlk** [comment]
> Not sure if moving around test case (like `MM-T2185`) is something that we should normally do. In my mind, move test either a passing/failing test into a new spec file to isolate but a fix should be made to the failing test on the same PR, unless such failed test is a known issue like product bug, test compatibility issue among server edition/license (Cloud, E20, TE) or a limitation in our test infra.
> 
> I'm good as is but open for discussion as a group.

Thanks @saturninoabril , yes you are right I misplaced the stable test case to wrong file.
everything is back to original file except T2183, it has moved to `custom_emoji_1` as it is stable.

**Review by prapti — APPROVED** [review]
Agree with Saturn's argument.
Looks good for now.
Thank you for fixing it @furqanmlk!

**Pull request merged**